backend/uclapi/dashboard/app_helpers.py

```diff
@@ -2,15 +2,26 @@
 from random import SystemRandom
 
 from common.helpers import generate_api_token
+from uclapi.settings import REDIS_UCLAPI_HOST
 
 import os
+import redis
 import textwrap
 import validators
 
 def generate_temp_api_token():
     return generate_api_token("temp")
 
 
+def get_temp_token():
+    r = redis.StrictRedis(host=REDIS_UCLAPI_HOST)
+
+    token = generate_temp_api_token()
+    # Limit is implemented inside throttle, so no value is required.
+    r.set(token, "LIMIT", 60 * 5)
```

Comment:
Is 5 minutes enough? Is the number of uses still limited as before?

Reply:
we throttle, which in turn is limit. so yeah

Reply:
Instead of implementing a limit like this, we should set the limit in code to a maximum number of requests (e.g. 10 as we have now), and set the token to expire using a Redis TTL.